Gemfile

```diff
@@ -10,6 +10,14 @@ source 'http://repo.release.cerner.corp/internal/rubygems/' if internal
 gem 'berkshelf', '~> 3.0', group: :deployment
 gem 'rubocop', '~> 0.18'
 gem 'foodcritic', '~> 4.0'
+gem 'rspec',  '~> 3.1'
+
+# https://github.com/opscode/chef/issues/2547
+if Bundler.current_ruby.on_19?
```

Comment:
Should get interesting soon.  Ruby 1.9.3 end of development was a year ago.  End of security patches is one month away: https://www.ruby-lang.org/en/news/2014/01/10/ruby-1-9-3-will-end-on-2015/ So this is a larger discussion than this issue.

For this issue :shipit: but expect conversations on the change.